File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/DatabaseBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class DatabaseBean {
    private String code;
    private String template;
    private Boolean fileShow;
    private Boolean defaultDatabase;
    private String createTableTemplate;
    private String deleteTableTemplate;
    private String rebuildTableTemplate;
    private String createFieldTemplate;
    private String updateFieldTemplate;
    private String deleteFieldTemplate;
    private String deleteIndexTemplate;
    private String createIndexTemplate;
    private String updateTableComment;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/DatabaseBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class DatabaseBean {
    var code: String? = null
    var template: String? = null
    var fileShow: Boolean? = null
    var defaultDatabase: Boolean? = null
    var createTableTemplate: String? = null
    var deleteTableTemplate: String? = null
    var rebuildTableTemplate: String? = null
    var createFieldTemplate: String? = null
    var updateFieldTemplate: String? = null
    var deleteFieldTemplate: String? = null
    var deleteIndexTemplate: String? = null
    var createIndexTemplate: String? = null
    var updateTableComment: String? = null
}
